### PR TITLE
Updated desktop file for the new format

### PR DESCRIPTION
**Pull request by jkonecny12**
Now it pass without warning or error through desktop-file-validate tester.

### CODE REVIEW

File: etc/PlayOnLinux.desktop [hunk 1]
```diff
@@ -1,9 +1,8 @@
 [Desktop Entry]
 Version=1.0
-Encoding=UTF-8
 Name=PlayOnLinux
-Comment=PlayOnLinux
-Type=Application %F
```
Comment (qparis) [inline etc/PlayOnLinux.desktop]: Why removing %F?

Comment (jkonecny12) [inline etc/PlayOnLinux.desktop]: I was done changes on behalf of desktop-file-validate utility and for this line I get following output:

`playonlinx.desktop: error: value "Application %F" for key "Type" in group "Desktop Entry" is not a registered type value ("Application", "Link" and "Directory")`

I'm not quite expert on *.desktop files so it's possible I have fixed it wrong.

Comment (jkonecny12) [inline etc/PlayOnLinux.desktop]: The `%F` was add to Exec where it should be I think.

**Comment by jkonecny12** [comment]
Updated:
Fix `%F` I think now it is on good place.
Fix the categories.

**Comment by jkonecny12** [comment]
Thank you for the merge! :smile:

**Comment by ShapeShifter499** [comment]
Can someone please explain what %F is for?

**Comment by jkonecny12** [comment]
https://developer.gnome.org/integration-guide/stable/desktop-files.html.en

**Pull request merged**